man/sbd.8.pod

```diff
@@ -333,6 +333,22 @@ prevent a successful crashdump from ever being written.
 
 Defaults to 240 seconds. Set to zero to disable.
 
+=item B<-r> I<N>
+
+Actions to be executed when sbd inquisitor loop detects a timeout or
+one of the watchers detects it's parent to have died.
+
+Set reboot-action to comma-separated combination of
+noflush|flush plus reboot|crashdump|off.
+If just one of both is given the other stays at the default.
+
+This doesn't affect actions like off, crashdump, reboot explicitly
+triggered via mailbox messages.
+And it does as well not configure the action a watchdog would
+trigger should it run off (there is no generic interface).
+
+Defaults to flush,reboot.
```

Comment:
deliberately omitted that reference here as we have just this very short section in the man-page about sysconfig (2 variables + reference to template) otherwise and everything else is in the template.
the precidence of cmd-line-parameters over sysconfig is the same as with the other parameters although I think it is nowhere made explicit - neither man-page nor template - a fact that should probably be changed.

Reply:
that makes sense